phys2bids/interfaces/txt.py

```diff
@@ -10,10 +10,64 @@
 import numpy as np
 
 from phys2bids.physio_obj import BlueprintInput
+from operator import itemgetter
+from collections import Counter
```

Comment:
These should be added to the requirements if they're really necessary.

Reply:
We can do that

Reply:
No they shouldn't, if they are core python packages

Reply:
Are they?

Reply:
That is something you need to tell us - did you install the packages? Can you check?

Reply:
So i didn't install anything on travis, that should mean at least that the packages are required by the packages we have already as requisites

Reply:
operator is a core module, and collections too you can found them here:
https://docs.python.org/3/library/

Reply:
No need to install

Reply:
Right - but you still need to check because if their support is dropped then we do need to add them to the requisites.
i quickly checked and they are core python modules. But please do recheck.

Reply:
What do you want me to recheck?. I didn't installed them in travis and they are in the core library of python3

Reply:
Yes don't worry - the message was posted after you posted your answer, but I couldn't see it due to the page refresh. You did recheck - it's good.